src/main/resources/plugin.yml

```diff
@@ -1,9 +1,10 @@
 name: TradeShop
-version: 1.8.1
+version: 2.0.0-DEV
 main: org.shanerx.tradeshop.TradeShop
+api-version: 1.13
+load: POSTWORLD
```

Comment:
I am not 100% sure we need this, but okay.

Reply:
the load we more then likely don't need but we don't need to load pre world so i would prefer we load after it, just in case. The API version we do need, Spigot doesn't like it otherwise since we are using 1.13 material names and not the legacy ones

Reply:
Okay, was only referring to the load clause.